src/main/java/duke/customClass/TaskList.java

```diff
@@ -0,0 +1,57 @@
+package duke.customClass;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
+
+import java.util.List;
+import java.util.ArrayList;
```

Comment:
Perhaps the import statements for java packages should come before those of third party packages? I noticed this in the imports of other files as well so it might be good to take a look :)